### ISSUE TITLE
Block requests to ngrok.io 

### ISSUE THREAD

**Issue by emilyemcqueen** [fishtown-analytics/terraform-aws-dbt-cloud-single-tenant#28]
TLDR: please add ngrok.io to our block list

**The rationale for request/documentation for posterity:** 
Guard Duty reported several connections to hostname 2.tcp.ngrok[.]io - an activity that's associated with malicious behavior ranging from spam to keylogging. 

Alert - You have a severity 8 GuardDuty finding type Backdoor:EC2/C&CActivity.B!DNS for the EC2 instance i-05a84ab99d05f4557 in the region us-east-1 as the EC2 instance i-05a84ab99d05f4557 is querying a domain name associated with a known Command & Control server. on the port . The first attempt was on 2020-12-23T15:07:05Z and the most recent attempt on 2021-01-05T08:43:48Z . The total occurrence is 4. For more details open the GuardDuty console at https://console.aws.amazon.com/guardduty/home?region=us-east-1#/findings?search=id%3D3ebb4833684106991ece665b8a112cb5”

user: bogdan.cojocar from Beautypie Teams
user: jbasbagill from Flexport Teams

Support will reach out individually to these users, but we would like to prevent future occurrences at the network level. 

**Considerations** 
Historically, we've seen users leverage ngrok.io, an anonymizing service, to tunnel to databases running locally during trials. Although this isn't malicious, it's not a best practice and alternative methods (ex: SSH tunnel) are available to accomplish the same goal. Therefore, the benefit to users is not worth the risk to our platform and we've decided to block the domain entirely.
